tests/expr_and_series/rank_test.py

```diff
@@ -166,10 +166,10 @@ def test_lazy_rank_expr(
     ):
         request.applymarker(pytest.mark.xfail)
 
-    if (
-        any(x in str(constructor) for x in ("pyspark", "duckdb"))
-        and method in {"average", "max", "ordinal"}
-    ) or ("duckdb" in str(constructor) and DUCKDB_VERSION < (1, 3)):
+    if "pyspark" in str(constructor) and method in {"average", "max", "ordinal"}:
+        request.applymarker(pytest.mark.xfail)
+
+    if DUCKDB_VERSION < (1, 3):
```

Comment:
CI is failing with XPASS, which means that a test is passing but it's marked as a failing one. That's because this condition will be entered for any backend - it should be:

```suggestion
    "duckdb" in str(constructor) and DUCKDB_VERSION < (1, 3):
```